### ISSUE TITLE
Notes cut off when pressed in a row

### ISSUE THREAD

**Issue by Demozo** [otanim/virtual-piano#2]
If you press a note twice quickly one after another, the first note cuts off.
I assume this is because you have a single audio source for every key on the piano?

**Comment by otanim** [comment]
@Demozo yes, you're right, I ain't have time to continue development and fixing bugs.
If you interested you can create a pull request of bug fixing and I'll accept it :)
I can give a hint to fix the issue: to make independent music playing piano should different audio tags, that means when you click on playing audio, it will create a new html audio tag. To avoid overload of audio tags that will be great to remove each audio tags when note music stops.
